EnglishTranslation-master/UI.tsv

```diff
@@ -2814,16 +2814,16 @@ UI_20170912_002813	{@st66}News	{@st66}공지
 UI_20170912_002814	{@st66}Notes	{@st66}소개글				
 UI_20170912_002815	News	공지글				
 UI_20170912_002816	Notes	소개글				
-UI_20170912_002817	Member {img %s 13 13}	길드원{img %s 13 13}				
+UI_20170912_002817	Name {img %s 13 13}	길드원{img %s 13 13}				
```

Comment:
this is not the translation, and furthermore, is fine as is; it's specifying who the member is